views/scss/admin.scss

```diff
@@ -2,7 +2,45 @@
 @import "inc/buttons";
 
 .permission-container {
-    table td {
-        min-width: 130px;
+    .grid-container {
+        margin-top: 40px;
+        margin-bottom: 20px;
+    }
+
+    .msg-edit-area {
+        border: 1px $uiGeneralContentBorder solid;
+        background: $canvas;
+        padding: 20px 20px 10px 20px;
+        @include simple-border();
+
+        .icon-add {
+            font-size: 2rem !important;
```

Comment:
I replaced the classic rules by the related mixins

2015-06-02 10:10 GMT+02:00 Bertrand CHEVRIER <notifications@github.com>:

> In views/scss/admin.scss
> <https://github.com/oat-sa/extension-tao-dac-simple/pull/15#discussion_r31500393>
> :
>
> > @@ -2,7 +2,45 @@
> >  @import "inc/buttons";
> >
> >  .permission-container {
> > -    table td {
> > -        min-width: 130px;
> > +    .grid-container {
> > +        margin-top: 40px;
> > +        margin-bottom: 20px;
> > +    }
> > +
> > +    .msg-edit-area {
> > +        border: 1px $uiGeneralContentBorder solid;
> > +        background: $canvas;
> > +        padding: 20px 20px 10px 20px;
> > +        @include simple-border();
> > +
> > +        .icon-add {
> > +            font-size: 2rem !important;
>
> There's a font-size (check for the exact name) mixin that you should use
> instead
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oat-sa/extension-tao-dac-simple/pull/15/files#r31500393>
> .
>



-- 




*Jean-Sébastien CONAN*
Application Architect

•   Email:   jean-sebastien.conan@vesperiagroup.com
•   GSM: (+352) 621 320 343

*Vesperia Group*
148, Route d'Arlon
L-8010 Strassen
Luxembourg

http://www.vesperiagroup.com/